src/main/java/reposense/report/SummaryReportJson.java

```diff
@@ -9,6 +9,8 @@
  * Represents the structure of summary.json file in reposense-report folder.
  */
 public class SummaryReportJson {
+    public static final String SUMMARY_JSON_FILE_NAME = "summary.json";
```

Comment:
Is this the only appearance of "summary.json" in our codebase?

Reply:
The "summary.json" is used twice: in source code `RepoSense.java` and the test code `FileUtilTest.java`

Reply:
I mean that, the `summary.json` here: 
https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/report/ReportGenerator.java#L238 can be replaced.

Reply:
Oh ok noted. Will replace that with the static variable.